src/Services/Search/ParametersHandler.php

```diff
@@ -71,14 +85,18 @@ public function handlePaginationAndSorting($search, $request)
      */
     public function getSortingOptions()
     {
-        return [
-            'item.score' => $this->getTranslator()->trans("Findologic::search.sortRelevance"),
-            'sorting.price.avg_asc' => $this->getTranslator()->trans("Findologic::search.sortPriceAsc"),
-            'sorting.price.avg_desc' => $this->getTranslator()->trans("Findologic::search.sortPriceDesc"),
-            'texts.name1_asc' => $this->getTranslator()->trans("Findologic::search.sortLabelAsc"),
-            'default.recommended_sorting' => $this->getTranslator()->trans("Findologic::search.sortSalesFrequency"),
-            'variation.createdAt_desc' => $this->getTranslator()->trans("Findologic::search.sortDateAdded")
-        ];
+        /** @var CeresConfig $config */
+        $config = $this->getConfig();
+
+        $returnArray = [];
+
+        foreach ($config->sorting->data as $data) {
+            if (in_array($data, Plugin::API_SORT_ORDER_AVAILABLE_OPTIONS)) {
+                $returnArray[$data] = $this->getTranslator()->trans("Ceres::Template." . SearchOptions::TRANSLATION_MAP[$data]);
```

Comment:
`$returnArray[$data] = 'Ceres::Template.' . SearchOptions::TRANSLATION_MAP[$data];` does the same.